draft-ietf-httpbis-safe-method-w-body.xml

```diff
@@ -109,17 +109,13 @@ Host: example.org
 </artwork>
 
     <t>
-      if the query parameters extend to several kilobytes or more of
-      data it may not be, because many implementations place limits on their size. Often these limits are not
-      known or discoverable ahead of time, because a request can pass through many uncoordinated
-      systems. Additionally, expressing some data in the target URI is inefficient, because it
-      needs to be encoded to be a valid URI.
-    </t>
-
-    <t>
-      Encoding query parameters directly into the request URI also effectively
-      casts every possible combination of query inputs as distinct
-      resources. Depending on the application, that may not be desirable.
+      However, for a query with parameters that extend to several kilobytes or more of
+      data, encoding it in the request URI may not be the best option because 
+      <ul>
+        <li>many implementations place limitations on query size (and those limits are not discoverable ahead of time),</li>
+        <li>expressing some data in the target URI is inefficient because of the overhead of encoding that data into a valid URI, and</li>
+        <li>encoding query parameters directly into the request URI effectively casts every possible combination of query inputs as distinct resources.</li>
```

Comment:
Any point in an informative reference to the work on making query strings less of a caching disaster: https://github.com/jeremyroman/http-no-vary-search/  We shouldn't block on that, but if someone gets here, No-Vary-Search might still be an option for them.

Reply:
I believe that I properly incorporated an informative reference for this into the PR. I hope that I did it correctly!